node/pkg/cni/token_watch.go

```diff
@@ -162,6 +154,21 @@ func (t *TokenRefresher) Run() {
 	}
 }
 
+func (t *TokenRefresher) getSleepTime(nextExpiration *time.Time) time.Duration {
+	now := time.Now()
+	sleepTime := nextExpiration.Sub(now) / t.defaultRefreshFraction
+	if duration, err := time.ParseDuration(os.Getenv("CNI_TOKEN_REFRESH_DURATION")); err == nil {
```

Comment:
```suggestion
	if duration, err := time.ParseDuration(os.Getenv("CNI_TOKEN_REFRESH_INTERVAL")); err == nil {
```

A bit of a nit, but INTERVAL is a more appropriate term here - we're measuring the time between two events, not the length of a singular event.